pandas/core/common.py

```diff
@@ -441,13 +441,22 @@ def _get_callable_name(obj):
     return None
 
 
-def _apply_if_callable(maybe_callable, obj, **kwargs):
+def _apply_if_callable(maybe_callable, obj, axis=None, **kwargs):
```

Comment:
is the `axis=None` a left-over from the previous version of the PR? (it's not used anymore)

Reply:
no used, but its ok to put in here

Reply:
sorry, I fail to see how it is useful. It is just adding an unused argument, complicating the code?